do-dont/src/Dont.ts

```diff
@@ -1,18 +1,34 @@
-import { html } from 'lit';
+import { html, css } from 'lit';
 import { InstructionsCard } from './InstructionsCard';
 
 /**
  * Show cautions in general or for a specific component.
  */
 export class Dont extends InstructionsCard {
-  color = '#DE350B';
-  icon = (color: string) => html`
+  instanceStyle = css`
+    :host {
+      --dockit-dont-color: #de350b;
+    }
+    :host div.container div.instructions-container {
+      border-top-color: var(--dockit-dont-color);
+    }
+    :host div.container div.instructions-container div.background {
+      background-color: var(--dockit-dont-color);
+    }
```

Comment:
there is no need to write such selectors
just use classes to target element, that's it (also in some other files I saw overuse of nested selectors and `tag.class` selectors where single classes were just good)

```
    :host {
      --dockit-dont-color: #de350b;
    }
    .instructions-container {
      border-top-color: var(--dockit-dont-color);
    }
    .background {
      background-color: var(--dockit-dont-color);
    }
```